src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -189,6 +187,18 @@ class WorkspaceService(dataSource: DataSource, workspaceDAO: WorkspaceDAO, entit
       }
     }
 
+  // TODO: should I stay or should I go (move to a different class)?
+  //
+  // STAY: needs dataSource, workspaceDAO, and gcsDAO which are available here
```

Comment:
Since we need to also protect entity and other services, id assume this would need to go somewhere else